test/module/irohad/ametsuchi/wsv_query_command_test.cpp

```diff
@@ -33,12 +31,15 @@ namespace iroha {
     class WsvQueryCommandTest : public AmetsuchiTest {
      public:
       WsvQueryCommandTest() {
-        domain.domain_id = "domain";
-        domain.default_role = role;
-        account.domain_id = domain.domain_id;
-        account.account_id = "id@" + account.domain_id;
-        account.quorum = 1;
-        account.json_data = R"({"id@domain": {"key": "value"}})";
+        domain = clone(
+            TestDomainBuilder().domainId("domain").defaultRole(role).build());
+
+        account = clone(TestAccountBuilder()
+                            .domainId(domain->domainId())
+                            .accountId("id@" + domain->domainId())
+                            .quorum(1)
+                            .jsonData(R"({"id@domain": {"key": "value"}})")
```

Comment:
You use `id@domain` 11 times, I would suggest to define a constant, same for the whole string, you wrote it 3 times

Reply:
Yes, but it is a part of JSON. If I use a constant, it would reduce JSON readability.